src/BuildIntegration/Microsoft.NETCore.Native.Unix.props

```diff
@@ -63,6 +63,11 @@ See the LICENSE file in the project root for more information.
       <AdditionalNativeLibrary Include="$(IlcPath)/framework/System.Security.Cryptography.Native.OpenSsl.a" Condition="'$(TargetOS)' != 'OSX'"/>
     </ItemGroup>
 
+    <ItemGroup Condition="'$(TargetOS)' == 'OSX'">
```

Comment:
I would append these directly onto the linker command line since this is OSX special case:

```
<LinkerArg Include="-framework CoreFoundation" />
<LinkerArg Include="-framework Security" />
```

If you would like to keep the ItemGroup for it for some reason, I would call it `AdditionalNativeOSXFramework` or something similar so that it is clear that it is native part specific to OSX.

Reply:
> I would append these directly onto the linker command line since this is OSX special case

I was looking at it the same way as `AdditionalNativeLibrary`: if we need this for our own things, maybe outside developers will need this for their own things, and it should be extensible the same way. If someone can use CoreRT with e.g. GLUT framework on OSX to build OpenGL UI, more power to them.

> I would call it `AdditionalNativeOSXFramework` or something similar

This is all ignored outside OSX, so I don't see much harm in the less verbose name, but if you really want me to rename it, I can rename it.

Reply:
Just my 2 cents.. We are currently appending OSX frameworks as LinkerArg's directly. However it's implemented it's gonna be important to maintain control over those arguments. The way it currently is, we have no way of overriding the default LinkerArgs that CoreRT sets, we can only add to them. Please have that in mind once you start adding default frameworks to link against.
Also, being able to specify them as a separate ItemGroup would be great for readability in our project file.

Reply:
> I was looking at it the same way as AdditionalNativeLibrary

I think this is more similar to ICU that we are adding to linker args directly `<LinkerArg Include="-licucore" Condition="'$(TargetOS)' == 'OSX'" />` . Native part that are required by netstandard/netcoreapp implementation.

For rare cases that want to have a fine grained control over what is referenced on the linker command line, I would add "DefaultNativeLibraries" property. If it is set to false, we will not add any of our defaults.

Reply:
> I don't see much harm in the less verbose name

The default meaning of framework in .NET world is managed framework - like ASP.NET, WCF, ... . I think the name needs to make it clear that it is a framework from different world. Maybe having OSX is too much, but I think it should  be at least `AdditionalNativeFramework`.

Reply:
Makes sense - fixed!